src/richText.js

```diff
@@ -424,12 +422,12 @@ define([
      */
     function makeBubble(form, xpath) {
         function _parseXPath(xpath, form) {
-            if (CASE_REF_REGEX.test(xpath)) {
+            if (!FORM_REF_REGEX.test(xpath)) {
```

Comment:
That's correct. The spec shows the same icons for case and user properties. @dimagi/product might have more comments on this?